### DISCUSSION TITLE
How to configure VS code?

### DISCUSSION THREAD

**Discussion by swetepete** [coder/code-server#5061] [Q&A] [answered]
Do I enter VS Code configuration commands into code-server/config.yaml?

Could anyone provide an example?

Thank you

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
No, fpr global configs, enter JSON values at settings.json. To open that, hit F1 on your keyboard, type "Settings (JSON)" and open "Preferences: Settings(JSON)". There, place your custom json values and you are good to go :)
